data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+	add attributes
+		"bunks" -25
+	outfits
+		"Liquid-Class Shield"
+		"Metallic Hydrogen Cell"
+		"Cloud Piercer" 20
+
+		"MH Red-Class Engine"
+		"MH Red-Class Steering"
+
+mission "Incipias: The Derelict"
+	landing
+	invisible
+	to offer
+		has "Visit Quarg in Incipias Space: offered"
+	npc assist
+		government "Hicemus"
+		ship "Nimbus FTL" "HIC Veni #000001"
+		system "Hui'uc"
+		to despawn
+			has "incipias: helped the derelict"
+		on assist
+			conversation
+				`The derelict ship hails you as you approach it. Once you accept the hail, color gradients start moving over your screen in a dizzying fashion - curiously, your speakers remain silent. It soon becomes clear that you will not be able to understand whatever transmission you just witnessed without some sort of translator. You continue your boarding maneuver.`
+				scene "scene/incipias message"
+				`	You manage to dock to the disabled ship quite easily. The hatch opens and a hallway opens up before you. It is immediately obvious that this ship was not designed for humans; the walls are strangely curved and the floor has nearly no grip at all. As you carry the necessary repair equipment into the ship, your view is caught by five giant aliens half octopode, half jellyfish in appearance. They pulsate in bright colors similar to the gradients that appeared on your screen earlier. One of them is floating behind a giant mounted gun, and the two barrels are pointed right at you. The gunner flashes in an orange light, and you can feel the tension in the room rising.`
+				choice
+					`	(Try to convey that I intend no harm.)`
+						goto peace
+					`	(Drop the repair equipment and run back into my ship.)`
+						goto flee
+					`	(Try to make a non-lethal shot at the one behind the turret and run.)`
+				`	You hit the creature manning the turret, causing it to explode with a strident sound. Before you can enjoy your short-lived victory, you notice a motion further down the hallway. Turning your head you see a gunner flashing in a deep red, before shooting you down and avenging their fallen comrade.`
+					die
+				label peace
+				`	Feeling that the situation could escalate any moment, you slowly place the crate with the repair equipment on the ground and walk back a few steps. You are careful not to make any hasty movements. Both sides are still for a moment until one of the aliens slowly floats forward. Tones of yellow are now mixing into the creature's orange color, and two of their tentacles wrap around the crate. The other four back off in response. Once the alien manages to open the crate and look inside, its colors shift to a weak green. The rest of the crew also seem to relax as they take on the same color.`
+				`	With the repair equipment delivered, there is nothing else you can do, given the communication barrier. Just as you leave you can see how the aliens start pulsating in all colors of the spectrum; gradients are moving over their skin so fast that they almost look white.`
```

Comment:
```suggestion
				`	With the aliens retrieving the repair equipment, there is nothing else you can do, given the communication barrier. Just as you leave, you can see how the aliens start pulsating in all colors of the spectrum; gradients are moving over their skin so fast that they almost look white.`
```
This wasn't really a "delivery"; you wanted to help them, got scared, and then left the equipment.